cmd/erasure-healing.go

```diff
@@ -380,6 +381,9 @@ func (er *erasureObjects) healObject(ctx context.Context, bucket string, object
 	dryRun := opts.DryRun
 	scanMode := opts.ScanMode
 
+	if versionID == nullVersionID {
+		versionID = ""
+	}
```

Comment:
this may not be correct for versioned buckets?

Reply:
@harshavardhana I can't tell. Removing it.

Reply:
I will remove it, but someone should really check this and all uses of it below.